gitbook-docs/streaming_api.md

```diff
@@ -46,6 +47,7 @@ The server SHOULD provide:
 The server MAY provide:
 - `self` is the unique identifier of the vessel using the URN format specified for the uuid field in the Signal K schema. It may also use the URN format specified for the mmsi field in the Signal K schema if it exists. This is only provided if the server relates to a specific vessel, aircraft, aid to navigation or sar.
 - `name` is the name of the Signal K server software, e.g. signalk-server
+- `authenticationRequired` indicates when when the server requires authentication. The value can be `always`, `forwrite` or `never`.
```

Comment:
Since older versions of SK spec did not include security wouldn't it make more sense to assume that no security requirement is indicated by `authenticationRequired` not being there?

Reply:
Actually do we need `authenticationRequired` at all? In #508 we have result codes being sent, these can include 403 Authentication Required. Why not just use that?